profiles/common/modules/features/nexteuropa_editorial/nexteuropa_editorial.module

```diff
@@ -114,3 +132,94 @@ function nexteuropa_editorial_module_implements_alter(&$implementations, $hook)
     $implementations['nexteuropa_editorial'] = $group;
   }
 }
+
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function nexteuropa_editorial_form_workbench_moderation_moderate_form_alter(&$form, &$form_state, $form_id) {
+  $entity = $form['node']['#value'];
+
+  if (isset($entity->tracking_activate)) {
+    if (!isset($form['language'])) {
+      // Aligning the language setting for validate callback.
+      $form['tracking_activate'] = array(
+        '#type' => 'value',
+        '#value' => $entity->tracking_activate,
+      );
+      $language = entity_language('node', $entity);
+      $form['language'] = array(
+        '#type' => 'value',
+        '#value' => $language,
+      );
+    }
+    $form['#validate'][] = '_nexteuropa_editorial_tracking_wbm_validate';
+  }
+}
+/**
+ * Implements hook_field_widget_WIDGET_TYPE_form_alter().
+ *
+ * It implements it for "Text area" widget.
+ */
+function nexteuropa_editorial_field_widget_text_textarea_form_alter(&$element, &$form_state, $context) {
+  _nexteuropa_editorial_field_widget_tracking_form_alter($element, $form_state, $context);
+}
+
+/**
+ * Implements hook_field_widget_WIDGET_TYPE_form_alter().
+ *
+ * It implements it for "Text area with a summary" widget.
+ */
+function nexteuropa_editorial_field_widget_text_textarea_with_summary_form_alter(&$element, &$form_state, $context) {
+  _nexteuropa_editorial_field_widget_tracking_form_alter($element, $form_state, $context);
+}
+
+/**
+ * Implements hook_field_widget_WIDGET_TYPE_form_alter().
+ *
+ * It implements it for "Text field" widget.
+ */
+function nexteuropa_editorial_field_widget_text_textfield_form_alter(&$element, &$form_state, $context) {
+  _nexteuropa_editorial_field_widget_tracking_form_alter($element, $form_state, $context);
+}
+
+/**
+ * Implements hook_entity_load().
+ */
+function nexteuropa_editorial_entity_load($entities, $type) {
+  foreach ($entities as $entity) {
```

Comment:
Ok for checking the node type but not for the use of the arg() because I ensured that the type is present in the entity attached to the form. I tried to limit its use when I deal with forms because they can be embedded by custom or contrib modules into other pages.Then, I have no trust in it.